### PR TITLE
Automate sending flight wrapup emails

### PR DESCRIPTION
**Pull request by davidfischer**
This will send them directly rather than leaving them as drafts.

Currently, our production Front Backend settings use @ericholscher's account with the "name" set to `EthicalAds by Read the Docs`. These can be customized in our private -ops repo. We could also set a custom name *just* for flight wrapups if we want to (eg. "EthicalAds Flight Tracker") with a couple modifications.

### CODE REVIEW

**Review by ericholscher — COMMENTED** [review]
I feel like we want to modify the email content here, along with automating it.

Also I _love_ `EthicalAds Flight Tracker` as a name :)

**Comment by davidfischer** [comment]
> I feel like we want to modify the email content here, along with automating it.

What content are you thinking of changing?

**Comment by ericholscher** [comment]
Looking at it again, I think it's probably fine... I thought I wanted changes, but didn't have anything obvious except small tweaks. I do feel like the language avoiding pluralization is still a bit awkward, but not sure how to make it better.

File: adserver/tasks.py [hunk 1]
```diff
@@ -729,7 +729,10 @@ def notify_of_completed_flights():
                 ],
             }
 
-            with mail.get_connection(settings.FRONT_BACKEND) as connection:
+            with mail.get_connection(
+                settings.FRONT_BACKEND,
+                sender_name=f"{site.name} Flight Tracker",
```
Comment (ericholscher) [inline adserver/tasks.py:734]: 😍

**Review by ericholscher — APPROVED** [review]

**Comment by davidfischer** [comment]
I switched it to use "EthicalAds flight tracker" as the sender name. The sender name is *not* used if you send a draft, however, since once somebody clicks send they take over as the sender.

File: adserver/tasks.py (hunk 1 above)
Comment (davidfischer) [reply adserver/tasks.py:734]: Screenshot from 2023-02-07 15-45-43

**Comment by ericholscher** [comment]
Yep, works the same with all our EA automation (eg. publisher emails)

**Pull request merged**